chore: husky installed, configured in package.json

gochicus opened the pull request:
ISSUES CLOSED: #31

gochicus committed:
chore: husky installed, configured in package.json

ISSUES CLOSED: #31

ad21ewda1231dsa123das12 reviewed (changes requested):
Это всё хорошо, а где результат внедрения?

gochicus commented:
> Это всё хорошо, а где результат внедрения?

Можешь чуть прояснить? Не совсем понял что именно требуется,  я сделал тестовый коммит с ошибкой, он отрабатывает. Рефактор пойдет отдельным заданием.

themetronome commented:
> > Это всё хорошо, а где результат внедрения?
> 
> Можешь чуть прояснить? Не совсем понял что именно требуется, я сделал тестовый коммит с ошибкой, он отрабатывает. Рефактор пойдет отдельным заданием.

https://github.com/Atlantis-Academy/frontend-study-gochicus/pull/32#pullrequestreview-574941055

gochicus committed:
refactor: refactor according rules in eslint

ISSUES CLOSED: #31

ad21ewda1231dsa123das12 reviewed (changes requested):
Почему semicolon'ы проставлены? Точно подцепился нужный конфиг?

gochicus commented:
> Почему semicolon'ы проставлены? Точно подцепился нужный конфиг?

Кажется, husky слетел.  Ещё потестирую сегодня.

gochicus committed:
refactor: .eslintrc.json file's names refactored

affects: @lessons/how-to-js, @lessons/how-to-use-functions, @lessons/how-to-use-objects,
@lessons/loops-and-conditions, @practice/actions-with-page, @practice/events-in-page,
@practice/food-delivery

ISSUES CLOSED: #31

gochicus committed:
style: eslint rules implementation in code

affects: @lessons/how-to-js, @lessons/how-to-use-functions, @lessons/how-to-use-objects,
@lessons/loops-and-conditions, @practice/actions-with-page, @practice/events-in-page,
@practice/food-delivery

ISSUES CLOSED: #31

gochicus committed:
chore: excessive dependency removed

ISSUES CLOSED: #31

TorinAsakura commented on practice/food-delivery/src/index.ts:
```diff
@@ -1,88 +1,80 @@
     }
-  });
+  })
   // Countdown timer food delivery
```
комменты в формате JSDoc должны быть

TorinAsakura commented on practice/events-on-page/src/index.ts:
```diff
@@ -1,60 +1,58 @@
+  const form: HTMLFormElement = document.querySelector('form.add')
+  const input: HTMLInputElement = form.querySelector('.adding__input')
+  const checkbox: HTMLInputElement = form.querySelector("[type='checkbox']")
+  const bin: NodeListOf<Element> = document.querySelectorAll('.delete')
```
что такое bin?

gochicus committed:
refactor: variable name changed, commentary format changed

affects: @practice/events-in-page, @practice/food-delivery

ISSUES CLOSED: #31

TorinAsakura commented on practice/events-on-page/src/index.ts:
```diff
@@ -1,60 +1,58 @@
-      item.parentElement.remove();
-      movieDB.movies.splice(i, 1);
-      createMoviesList(movieDB.movies, ul);
-    });
-  });
+  binIcon.forEach((item, i) => {
```
Да бляха! Почему удаление это BIN-то?! Что за жесть?! Почему не назвать просто remove без выпендрёжей?!

gochicus committed:
refactor(events-on-page): variable renamed

affects: @practice/events-in-page

ISSUES CLOSED: #31
